package.json

```diff
@@ -39,10 +39,13 @@
     "expo": "39.0.0",
     "expo-asset": "8.2.0",
     "expo-constants": "9.2.0",
+    "expo-document-picker": "~8.4.0",
```

Comment:
Please read contributing file : https://github.com/NMF-earth/nmf-app/blob/master/contributing.md => no ~ in package.json